domains/boldbydesign.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "For Business EMail",
+    "repo": "https://github.com/boldbydesign",
+    "owner": {
+        "username": "boldbydesign",
+        "email": "jeremiah.helfer@protonmail.com"
+    },
+    "record": { 
+        "MX": ["boldbydesign.is-a.dev, -0"],
```

Comment:
`-0` is not a valid MX record value, the array of MX records only allows record values, not priority.